core/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'bintray-release'
 
 dependencies {
-    compile 'com.android.support:support-v4:22.1.1'
+    compile 'com.android.support:design:22.2.0'
```

Comment:
:x: this will cause proguard issues for people who build release flavors with this library. There's a workaround:

https://github.com/ataulm/wutson/commit/6113a8996daf0174460201e5ec2f55b87ecc5168